_uw-research-computing/htc-job-file-transfer.md

```diff
@@ -0,0 +1,101 @@
+---
+highlighter: none
+layout: file_avail
+title: Data Storage Locations on the HTC
```

Comment:
"Using Data in Jobs" ? 

"Using Data Storage Locations on the HTC System"? 

It's nice to have a verb.

Reply:
Changed to "Using and transferring data in jobs on the HTC system"